src/main/java/com/eleks/academy/whoami/core/state/ProcessingQuestion.java

```diff
@@ -36,4 +38,14 @@ public String getCurrentTurn() {
 		return this.currentPlayer;
 	}
 
+	private void updatePlayerStates() {
```

Comment:
for now, as we see that this action about setting the state to players happens only in the constructor and nowhere else, let's keep it there.
Private methods mostly are needed when there is some common code within the class that repeats or an action that provides a single result takes many lines of code to be reached. In such cases, private methods are needed, as you may see here we do not have such cases - so no need of private method